common/test/data/unicode_graded/README.md

```diff
@@ -0,0 +1 @@
+This is a very simple course for testing student responses reports for courses with unicode information.
```

Comment:
Also we have a unicode test course - what was wrong with using that one? (just trying to see how this is distinct from the `test_unicode` course